src/resources/variable/VariableAlias.yaml

```diff
@@ -0,0 +1,35 @@
+post:
+  summary: 'Create a variable alias'
+  description: |
+    - Allows you to add an alias at the level defined in the request body on an existing variable having a higher scope, in order to customize its key.
+    - You have to specify a key in the request body and the scope and the parent id of the alias
+    - The system will create a new variable at the requested level with the same value as the one corresponding to the variable id in the path
+    - The response body will contain the newly created variable
+    - Information regarding the aliased_variable will be exposed in the "aliased_variable" or in the "aliased_secret" field of the newly created variable
+    - Only 1 alias level is allowed. You can't create an alias on an alias
+  operationId: createVariableAlias
+  parameters:
+    - $ref: '../../parameters/path/variableId.yaml'
+  tags:
+    - Variable Main Calls
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: '../../schemas/variable/VariableAliasRequest.yaml'
+  responses:
+    '201':
+      description: 'Create variable alias'
+      content:
+        application/json:
+          schema:
+            $ref: '../../schemas/variable/VariableResponse.yaml'
+    '400':
+      description: Can't create an alias on a higher scope. Aliases can only be created from one scope to a lower scope. Scope hierarchy is BUILT_IN > PROJECT > ENVIRONMENT > CONTAINER
```

Comment:
Scope hierarchie is 
ORGANIZATION > PROJECT > ENVIRONMENT > Service

BUILT_IN are variables created by Qovery, but they still have a scope